examples/client_example/client_example.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+"""Python Client Example."""
+
+# Copyright 2012 - 2017, New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+import argparse
+import logging
+import os
+import shutil
+from pathlib import Path
+
+from tuf.exceptions import RepositoryError
+from tuf.ngclient import Updater
+
+# constants
+BASE_URL = "http://127.0.0.1:8000"
+DOWNLOAD_DIR = "./downloads"
+METADATA_DIR = f"{Path.home()}/.local/share/python-tuf-client-example"
+CLIENT_EXAMPLE_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def init():
+    """
+    The function that initializes the TUF client infrastructure.
+
+    It creates the metadata directory and adds a trusted ``root.json``.
+    """
+
+    if not os.path.isdir(DOWNLOAD_DIR):
+        os.mkdir(DOWNLOAD_DIR)
+        print(f"Download directory [{DOWNLOAD_DIR}] was created")
+
+    if not os.path.isdir(METADATA_DIR):
+        os.makedirs(METADATA_DIR)
+        print(f"Metadata folder [{METADATA_DIR}] was created")
+
+    if not os.path.isfile(f"{METADATA_DIR}/root.json"):
+        shutil.copy(
+            f"{CLIENT_EXAMPLE_DIR}/1.root.json", f"{METADATA_DIR}/root.json"
+        )
+        print(f"Added trusted root in {METADATA_DIR}")
+
+    else:
+        print(f"Found trusted root in {METADATA_DIR}")
+
+
+def download(target):
+    """
+    The function that downloads the target file using the TUF ``nglcient``
+    Updater.
```

Comment:
Fixed a typo, made this more in line with the style guide:
```suggestion
    """Download the target file using ``ngclient`` Updater.
```